CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added measure reporting to ANC application
```

Comment:
Please add the module name separated by a pipe character as a prefix to all change log entries